lms/djangoapps/verify_student/views.py

```diff
@@ -1162,48 +1163,46 @@ def results_callback(request):
         return HttpResponseBadRequest("edX ID {} not found".format(receipt_id))
 
     user = attempt.user
-    email_config = EmailMarketingConfiguration.current()
     verification_status_email_vars = {
         'platform_name': settings.PLATFORM_NAME,
     }
-    email_context = {
-        'email_config': email_config,
-        'email': user.email
-    }
     if result == "PASS":
         log.debug("Approving verification for %s", receipt_id)
         attempt.approve()
         status = "approved"
-        if email_config.sailthru_verification_passed_template:
-            expiry_date = datetime.date.today() + datetime.timedelta(
-                days=settings.VERIFY_STUDENT["DAYS_GOOD_FOR"]
-            )
-            verification_status_email_vars['expiry_date'] = expiry_date.strftime("%m/%d/%Y")
-            verification_status_email_vars['subject'] = _("Your {platform_name} ID Verification Approved").format(
-                platform_name=settings.PLATFORM_NAME
-            )
-            verification_status_email_vars['full_name'] = user.profile.name
-            email_context['template'] = email_config.sailthru_verification_passed_template
-            email_context['template_vars'] = verification_status_email_vars
-            send_verification_status_email(email_context)
+        expiry_date = datetime.date.today() + datetime.timedelta(
+            days=settings.VERIFY_STUDENT["DAYS_GOOD_FOR"]
+        )
+        verification_status_email_vars['expiry_date'] = expiry_date.strftime("%m/%d/%Y")
+        verification_status_email_vars['full_name'] = user.profile.name
+        subject = _("Your {platform_name} ID Verification Approved").format(
+            platform_name=settings.PLATFORM_NAME
+        )
+        message = render_to_string('emails/passed_verification_email.txt', verification_status_email_vars)
+        from_address = configuration_helpers.get_value(
+            'email_from_address',
+            settings.DEFAULT_FROM_EMAIL
+        )
+        send_verification_status_email(subject, message, from_address, user.email)
```

Comment:
I think it should be `send_verification_status_email.delay(subject, message, from_address, user.email)`